File: 250323/약수가 세 개인 수/numbers-with-three-divisors.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int start = 0, end = 0, count_1 = 0, count_2 = 0;
    cin >> start >> end;
    for(int i = start; i <= end; i++) {
        count_1 = 0;
        for(int j = 1; j <= i; j++) {
            if(i % j == 0) {
                count_1++;
            }
        }
        if(count_1 == 3) {
            count_2++;
        }
    }
    cout << count_2;
    return 0;
}

File: 250323/약수가 세 개인 수/numbers-with-three-divisors.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int start = sc.nextInt();
        int end = sc.nextInt();
        int count_1 = 0;
        int count_2 = 0;
        for (int i = start; i <= end; i++) {
            count_1 = 0;
            for(int j = 1; j <= i; j++) {
                if(i % j == 0) {
                    count_1++;
                }
            }
            if(count_1 == 3) {
                count_2++;
            }
        }
        System.out.println(count_2);
        sc.close();
    }
}